descriptors/ModuleDescriptor-template.json

```diff
@@ -1207,6 +1207,7 @@
         "circulation-storage.loans.collection.get",
         "circulation.rules.loan-policy.get",
         "circulation.rules.request-policy.get",
+        "circulation.rules.overdue-fine-policy.get",
```

Comment:
Please can the lost item fee policy permission also be included, as that is likely also missing and will cause a similar issue.

Reply:
Done